pyqtgraph/parametertree/parameterTypes/__init__.py

```diff
@@ -36,9 +36,9 @@
 registerParameterType('color',     ColorParameter,       override=True)
 registerParameterType('colormap',  ColorMapParameter,    override=True)
 registerParameterType('file',      FileParameter,        override=True)
-registerParameterType('float',     SimpleParameter,      override=True)
+registerParameterType('float',     NumericParameter,      override=True)
```

Comment:
There appears to be 4 additional types added by this PR (date, group, table, tableview) but not registered here. Was that intentional? This PR is quite large, perhaps it might be better to hold off those new types to another PR?

Reply:
Yes sorry it's a mistake caused by the bad push that I did